src/error.rs

```diff
@@ -0,0 +1,33 @@
+use defines::AfError;
+
+pub type ErrorCallback = Fn(AfError);
+
+pub static mut HANDLE_ERROR: &'static ErrorCallback = &handle_error_general; 
+
+pub fn register_error_handler(callback: &'static ErrorCallback) {
+    unsafe {
+        HANDLE_ERROR = callback;
+    }
+}
+
+pub fn handle_error_general(error_code: AfError) {
+    match error_code {
+        AfError::SUCCESS            => {}, /* No-op */
```

Comment:
True, but it is conceivable that the user wants a different method of error handling at a different point in the program execution and we should be robust in upholding that